apps/cli/src/downloadCodemod.ts

```diff
@@ -111,6 +111,25 @@ export class CodemodDownloader implements CodemodDownloaderBlueprint {
 			}
 		}
 
+		if (config.engine === 'ast-grep') {
+			const yamlPath = join(directoryPath, 'rule.yaml');
+			const yamlData = await this._fileDownloadService.download(
```

Comment:
can it throw? Should we wrap it in try/catch?

Reply:
It shouldn't throwing, but adding a try catch is always a good idea. Will do it.